src/popup.svelte

```diff
@@ -64,43 +76,109 @@
   function invalidPumlServerUrl(invalidUrl: string): string {
     return `${invalidUrl} does not refer a valid plantUML serer`;
   }
+
+  function setAsDefault() {
+    inputUrl = Constants.defaultConfig.pumlServerUrl;
+    handleChangeServerUrlButtonClick();
+  }
 </script>
 
-<div id="popup">
-  {#if loading}
-    <p>loading...</p>
-  {:else}
-    <button class="puml-vis-toggle" on:click={handleToggleButtonClick}
-      >{config.extensionEnabled ? 'Disable' : 'Enable'} PlantUML visualization</button
-    >
-    <p class="puml-vis-server-url">server: {config.pumlServerUrl}</p>
-    <input class="puml-vis-server-url" bind:value={inputUrl} />
-    <p class="puml-vis-error">{errorMessage}</p>
-    <button class="puml-vis-server-url" on:click={handleChangeServerUrlButtonClick}
-      >Change server URL (https is required)</button
-    >
-    <img class="puml-vis-version" src={versionPumlSrc} alt="PlantUML version" />
-  {/if}
-</div>
+<Header />
+
+<ItemContainer>
+  <ItemLabel>Visualize PlantUML code</ItemLabel>
+  <Switch id="switch" on:change={handleToggleButtonClick} value={config.extensionEnabled} />
+</ItemContainer>
+
+<ItemContainer>
+  <ItemLabel>Server URL</ItemLabel>
+  <TextField
+    bind:value={inputUrl}
+    on:blur={handleChangeServerUrlButtonClick}
+    on:keypress={(event) => {
```

Comment:
[Comment]
To improve performance, a normal function would be better than a arrow function
But it's not a big problem

Reply:
I read this.
https://svelte.dev/tutorial/inline-handlers

Reply:
compiler is so smart